components/brave_rewards/resources/ui/components/adsBox.tsx

```diff
@@ -78,20 +78,23 @@ class AdsBox extends React.Component<Props, {}> {
 
   render () {
     let adsEnabled
+    let adsUIEnabled
     const { adsData } = this.props.rewardsData
 
     if (!adsData) {
```

Comment:
nit: can we simplify this if? So that we set default values as adsData is empty and then we just have if adsData and we set it how we want based on props